certbot-ci/certbot_integration_tests/certbot_tests/test_main.py

```diff
@@ -0,0 +1,27 @@
+import requests
+import urllib3
+
+
+def test_hello_1(request, worker_id):
+    assert request.config.acme_xdist['http_port'][worker_id]
+    assert request.config.acme_xdist['https_port'][worker_id]
```

Comment:
The test here is about the existence of an http_port and an https_port for a given worker_id, not about an assertion about the worker_id value.

But this is a temporary test, I put here to illustrate the correct behavior of the test framework. It will be replaced by real tests later.